packages/shared/models/task.js

```diff
@@ -0,0 +1,31 @@
+import mongoose from 'mongoose'
+
+const taskSchema = new mongoose.Schema({
+  id: String,
+  title: String,
+  description: String,
+  captionURL: String,
+  attachments: [
```

Comment:
create field "data" with type BSON for file content
https://docs.mongodb.com/manual/reference/bson-types/